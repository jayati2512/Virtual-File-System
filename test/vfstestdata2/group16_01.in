createvfs college 1000
mountvfs college
unmountvfs college
mountvfs college
makedir /IIITB/IIITBStudy/FirstSem DM
makedir /songs/bands/ark livingthings
addfile /IIITB/IIITBStudy first.txt /usr/include/syscall.h
listdir / 1 /tmp/abc.txt
makedir /IIITB/IIITBStudy/FirstSem/oneer DM
addfile /songs/bands second.txt /usr/include/wait.h
deletedir /songs/bands/linkinpark/livingthings
listdir / 1 /tmp/listdiroutput.txt
listfile /songs/bands/second.txt /tmp/new.txt
searchfile first.txt /tmp/search.txt
makedir /IIITB/IIITBStudy/FirstSem DM
listdir / 1 /tmp/listdiroutput.txt
searchfile second.txt /tmp/search1.txt
copyfile /songs/bands/second.txt /IIITB/IIITBStudy/first.txt
movefile /IIITB/IIITBStudy/first.txt /songs/second.txt
searchfile first.txt /tmp/search2.txt
searchfile second.txt /tmp/search2.txt
movedir /songs/bands /IIITB/IIITBStudy
deletedir /IIITB/IIITBStudy/bands/linkinpark/livingthings
makedir /ganguly/dqqq l
deletedir /ganguly/dqqq/l
listdir / 1 /tmp/listdiroutput.txt
unmountvfs college
