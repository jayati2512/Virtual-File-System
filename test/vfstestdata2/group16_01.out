createvfs_SUCCESS
mountvfs_SUCCESS
unmountvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
addfile_SUCCESS 
listdir_SUCCESS
makedir_SUCCESS
addfile_SUCCESS 
deletedir_FAILURE CANNOT_FIND_SPECIFIED_DIR
listdir_SUCCESS
listfile_SUCCESS 
searchfile_SUCCESS 1
makedir_FAILURE DIRECTORY_ALREADY_EXISTS
listdir_SUCCESS
searchfile_SUCCESS 1
copyfile_SUCCESS 
movefile_SUCCESS 
searchfile_SUCCESS 1
searchfile_SUCCESS 1
movedir_SUCCESS 
deletedir_FAILURE CANNOT_FIND_SPECIFIED_DIR
makedir_SUCCESS
deletedir_SUCCESS 
listdir_SUCCESS
unmountvfs_SUCCESS
